### PR TITLE
fix: Drop not considering timezone

### LINKED ISSUE #9836: Drop not considering timezone
Drop is 14m from now, but there's an additional 4h: 

image

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #9836

## Needs QA check

- @kodadot/qa-guild please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-03-22 at 11 10 17@2x

### CODE REVIEW

File: components/drops/utils.ts [hunk 1]
```diff
@@ -32,7 +32,10 @@ export function formatDropStartTime(
   return intlFormat(startTime, options, { locale })
 }
 
-export const fromatCETDate = (date: string, time: string = ''): Date => {
-  const t = time ? `T${time}` : ''
-  return new Date(`${date}${t}+01:00`)
+export const fromatCETDate = (date: string, time: string): Date =>
```
Comment (Jarsen136) [inline components/drops/utils.ts:35]: typo `formatCETDate`

Comment (hassnian) [reply components/drops/utils.ts]: fixed 20fa856

**Comment by yangwao** [comment]
cc @prury

**Comment by prury** [comment]
> cc @prury

let's merge and i'll test locally or close to drop time
hassnian teaching me some mocking tricks

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 9.02 USD/DOT ~ 3.326 $DOT
🧗 [16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD](https://polkadot.subscan.io/account/16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)
🔗 [0x436485cfd4594cba0dbdd00050d722519dac105852efe857cf410489f0c07bbe](https://polkadot.subscan.io/extrinsic/0x436485cfd4594cba0dbdd00050d722519dac105852efe857cf410489f0c07bbe)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**